pkg/reconciler/brokercell/brokercell.go

```diff
@@ -42,22 +66,85 @@ var _ bcreconciler.Finalizer = (*Reconciler)(nil)
 func (r *Reconciler) ReconcileKind(ctx context.Context, bc *intv1alpha1.BrokerCell) pkgreconciler.Event {
 	bc.Status.InitializeConditions()
 
+	// Reconcile ingress deployment and service
+	ingressArgs := r.makeIngressArgs(bc)
+	if _, err := reconciler.ReconcileDeployment(r.kubeClientSet, r.deploymentLister, resources.MakeIngressDeployment(ingressArgs)); err != nil {
+		r.Logger.Errorf("Failed to reconcile ingress deployment for \"%s/%s\": %v", bc.Namespace, bc.Name, err)
```

Comment:
I think the namespace and name of the reconciled object is already a field on the log message, so no need to include them here (and elsewhere).

My preference is to use the non-sugared logger, so the variables are typed and checked at compile time. Using the sugared logger means these are only checked at runtime.